serverless-plugin/default-config.yaml

```diff
@@ -94,6 +94,13 @@ alarms:
     CPUUtilization:
       Statistic: Average
       Threshold: 90
+  SNS:
```

Comment:
The defaults can be added to the configuration section of the project README also

Reply:
Yes, defaults have been added indeed. Please check default alarms in the line **276** and default dashboard the line **363** in the README